lib/facet_example_fetcher.rb

```diff
@@ -26,10 +27,18 @@ def fetch
   end
 
 private
-  # Fetch facet examples for a given field
   def fetch_for_field(field_name, facet_params)
     example_count = facet_params[:examples]
     example_fields = facet_params[:example_fields]
+    scope = facet_params[:example_scope]
+
+    if scope == :global
+      query = nil
+      filter = nil
```

Comment:
This branch isn't necessary.  `example_scope` isn't a required parameter to this object, so it's possible that neither branch of this conditional will execute.  In that case `fetch_by_slug` will be handed `nil` and `nil` anyway, same as if it was `:global`.  May as well not bother explicitly setting to `nil`, or if you're keen on being explicit, setting them to `nil` whenever the scope isn't `:query`.